garrysmod/lua/includes/extensions/client/panel/dragdrop.lua

```diff
@@ -461,8 +467,7 @@ function meta:DragMouseRelease( mcode )
 
 	dragndrop.Drop()
 
-	-- Todo.. we should only do this if we enabled it!
-	if ( gui.EnableScreenClicker ) then
+	if ( gui.EnableScreenClicker && !self.ReleaseCursor ) then

```

Comment:
Maybe it should check `if CLIENT` or `if not MENU_DLL`, not `if gui.EnableScreenClicker`?

Reply:
I think the former is more appropriate.